indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -307,7 +309,9 @@ def _process(self):
                            '{num} emails were sent.', num_emails_sent).format(num=num_emails_sent), 'success')
             return jsonify_data()
 
-        registrations_without_ticket = [r for r in self.registrations if r.is_ticket_blocked]
+        ticket_template = self.regform.get_ticket_template()
+        registrations_without_ticket = [r for r in self.registrations if (ticket_template.is_ticket and
+                                                                          r.is_ticket_blocked)]
```

Comment:
```suggestion
        template_is_ticket = self.regform.get_ticket_template()        .is_ticket
        registrations_without_ticket = [r for r in self.registrations if template_is_ticket and r.is_ticket_blocked]
```

It's a bit of a waste to iterate over all placeholders of the ticket template again and again for each registrant.

Reply:
nice catch!